.github/workflows/build_and_test.yml

```diff
@@ -0,0 +1,45 @@
+name: Build and test
+
+on: pull_request
+
+jobs:
+  build-and-test:
+    runs-on: ubuntu-latest
+    container: ros:foxy
+
+    steps:
+      - name: Check out repo
+        uses: actions/checkout@v2
+
+      - name: Clone dependency packages
+        run: |
+          mkdir dependency_ws
+          vcs import dependency_ws < build_depends.repos
+
+      - name: Install missing dependencies
+        run: |
+          sudo apt-get update
+          rosdep update
+          DEBIAN_FRONTEND=noninteractive rosdep install --ignore-src --rosdistro foxy -y \
+          --from-paths obstacle_stop_planner_refine \
+          --skip-keys=vehicle_info_util \
+          --skip-keys=autoware_planning_msgs \
+          --skip-keys=autoware_perception_msgs \
+          --skip-keys=autoware_debug_msgs
+
+      - name: Build
+        run: |
+          . /opt/ros/foxy/setup.sh
+          colcon build \
+            --catkin-skip-building-tests \
```

Comment:
Oh, I found one more thing.
We should delete `--catkin-skip-building-tests` because we should run tests and we don't use catkin anymore.

Reply:
Exactly. Fixed in c05225d.